eng/pipelines/templates/jobs/smoke-tests.yml

```diff
@@ -16,6 +16,10 @@ jobs:
         vmImage: ubuntu-18.04
       displayName: Check Smoke Test Eligibility
       steps:
+        - template: /eng/pipelines/templates/steps/use-node-version.yml
+          parameters:
+            NodeVersion: "14.x"
```

Comment:
Unless you specifically need Node 14, I think you should just import `globals.yml`, then you can call `use-node-version.yml` with no parameter and it will use the default set in `globals.yml`.  This way, when the default is updated your pipeline will also automatically update.